### PR TITLE
fix: Docs reveal version for 2.x and controls modes example 

### PR DESCRIPTION
**Pull request by Savokr**
Fixed proper reveal version for 2.x documentation and controls mode example.

### CODE REVIEW

File: documentation/yarn.lock [hunk 1]
```diff
@@ -10633,10 +10610,10 @@ rxjs@7.3.0, rxjs@^7.1.0:
   dependencies:
     tslib "~2.1.0"
 
-rxjs@7.5.5:
-  version "7.5.5"
-  resolved "https://registry.yarnpkg.com/rxjs/-/rxjs-7.5.5.tgz#2ebad89af0f560f460ad5cc4213219e1f7dd4e9f"
-  integrity sha512-sy+H0pQofO95VDmFLzyaw9xNJU4KTRSwQIGM6+iG3SypAtCiLDzpeG8sJrNCWn2Up9km+KhkvTdbkrdy+yzZdw==
+rxjs@7.5.4:
```
Comment (pramodcog) [inline documentation/yarn.lock:10613]: Why the version got downgraded?

**Review by pramodcog — CHANGES_REQUESTED** [review]

File: documentation/yarn.lock (hunk 1 above)
Comment (Savokr) [reply documentation/yarn.lock:10613]: Not sure why, it was done by yarn automatically

Comment (larsmoa) [reply documentation/yarn.lock:10613]: Revert changes to yarn.lock to be sure... @Savokr

Comment (Savokr) [reply documentation/yarn.lock:10613]: @larsmoa It was updated specifically to update reveal version because previously it was 2.2.2 instead of 2.3.0

Comment (pramodcog) [reply documentation/yarn.lock:10613]: Revert only the `rxjs` changes in the yarn.lock

**Pull request merged**